optimize/backend/src/it/java/io/camunda/optimize/service/cleanup/CleanupSchedulerIT.java.out

```diff
@@ -110,7 +110,6 @@
 //     return embeddedOptimizeExtension.getCleanupScheduler();
 //   }
 //
-//   @AllArgsConstructor
 //   @Builder
 //   @Data
```

Comment:
these are files that are commented out. The reason why I deleted some annotations in them is because they were in the way when I was organizing my work. It doesn't really matter if we change these files